starter-dev-backend/src/datasources/github-api.ts

```diff
@@ -19,20 +22,45 @@ export class GitHubAPI extends RESTDataSource {
     request.headers['authorization'] = this.context.token;
   }
 
+  async getOrgsMemberCount(login: string) {
+    return await this.get(`/orgs/${login}/members?per_page=100}`);
+  }
+
+  async getOrgRepos(login: string) {
+    return await this.get(`/orgs/${login}`);
+  }
+
+  // https://docs.github.com/en/rest/orgs/orgs?apiVersion=2022-11-28#list-organizations-for-a-user
+  async getOrgs(login: string) {
+    const data = await this.get(`/users/${login}/orgs`);
+    // return orgFormatter(data);
+    return data;
+  }
+
+  async getOwnerStarCount(login: string) {
+    return await this.get(`/users/${login}/starred`);
+  }
+
   // https://docs.github.com/en/rest/users/users?apiVersion=2022-11-28#about-the-users-api
   async getOwner() {
-    return await this.get('/user');
+    const data = await this.get('/user');
+    // return ownerFormatter(data);
+    return data;
   }
 
   // https://docs.github.com/en/rest/repos/repos?apiVersion=2022-11-28#get-a-repository
   async getRepo(owner: string, repoName: string) {
-    return await this.get(`/repos/${owner}/${repoName}`);
+    const data = await this.get(`/repos/${owner}/${repoName}`);
+    // return repoFormatter(data);
+    return data;
   }
 
   // https://docs.github.com/en/rest/repos/repos?apiVersion=2022-11-28#list-repositories-for-a-user
   async getRepos(username: string, perPage?: string) {
-    return await this
+    const data = await this
       .get(`/users/${username}/repos?sort=updated&per_page=${perPage}
     `);
+    // return repoFormatter(data);
```

Comment:
Suggestion: Either remove unused and commented out code or reference a ticket number if related to the aforementioned formatting and testing ticket.